src/cookbooks/client_guidelines.md

```diff
@@ -0,0 +1,93 @@
+# Client Implementation Guidelines
+## Feature Gates
+
+The preferred method for testing experimental features is to __land in-tree__ and ship behind a __feature gate__. New features go through the standard Firefox review, testing, and deployment processes, and are then enabled experimentally in the field using these gates.
+A new Feature Gate library for Firefox Desktop is now available. Read the documentation for it to get started.
```

Comment:
Do you have a link to that documentation handy?

Reply:
Oops, got lost in google docs transition :) Will fix thanks.